packages/jest-circus/src/event_handler.js

```diff
@@ -4,11 +4,13 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow strict-local
```

Comment:
why?

Reply:
If I leave strict-local there then it complains about importing expect. Not a flow expert but this was the cause.

Is there a particular reason jest-circus uses strict-local and no other package in the project does? I'm not sure the history there.

Reply:
Strict local should be preferred as it's stricter. 

If it complained about expect we should fix expect :)

#6724

Reply:
Not sure what that issue link means in this context?

Reply:
I meant #6274... So close!